google-cloud-binary_authorization-v1/proto_docs/google/cloud/binaryauthorization/v1/service.rb

```diff
@@ -0,0 +1,189 @@
+        #   @return [::String]
+        #     Required. The resource name of the project associated with the
+        #     {::Google::Cloud::BinaryAuthorization::V1::Attestor attestors}, in the format `projects/*`.
+        # @!attribute [rw] page_size
+        #   @return [::Integer]
+        #     Requested page size. The server may return fewer results than requested. If
+        #     unspecified, the server will pick an appropriate default.
+        # @!attribute [rw] page_token
+        #   @return [::String]
+        #     A token identifying a page of results the server should return. Typically,
+        #     this is the value of {::Google::Cloud::BinaryAuthorization::V1::ListAttestorsResponse#next_page_token ListAttestorsResponse.next_page_token} returned
+        #     from the previous call to the `ListAttestors` method.
+        class ListAttestorsRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Response message for [BinauthzManagementService.ListAttestors][].
+        # @!attribute [rw] attestors
+        #   @return [::Array<::Google::Cloud::BinaryAuthorization::V1::Attestor>]
+        #     The list of {::Google::Cloud::BinaryAuthorization::V1::Attestor attestors}.
+        # @!attribute [rw] next_page_token
+        #   @return [::String]
+        #     A token to retrieve the next page of results. Pass this value in the
+        #     {::Google::Cloud::BinaryAuthorization::V1::ListAttestorsRequest#page_token ListAttestorsRequest.page_token} field in the subsequent call to the
+        #     `ListAttestors` method to retrieve the next page of results.
+        class ListAttestorsResponse
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request message for [BinauthzManagementService.DeleteAttestor][].
+        # @!attribute [rw] name
+        #   @return [::String]
+        #     Required. The name of the {::Google::Cloud::BinaryAuthorization::V1::Attestor attestors} to delete, in the format
+        #     `projects/*/attestors/*`.
+        class DeleteAttestorRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request to read the current system policy.
+        # @!attribute [rw] name
+        #   @return [::String]
+        #     Required. The resource name, in the format `locations/*/policy`.
+        #     Note that the system policy is not associated with a project.
+        class GetSystemPolicyRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request message for
+        # {::Google::Cloud::BinaryAuthorization::V1::ValidationHelperV1::Client#validate_attestation_occurrence ValidationHelperV1.ValidateAttestationOccurrence}.
```

Comment:
`"ruby-cloud-service-override": "...ValidationHelperV1=ValidationHelper"` doesn't seem to be working in this output.